interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -366,6 +369,16 @@
 </div> <!-- end bottomLeft -->
 <div id="bigCal">
 [-php-]
+/* used in debugging
+    foreach ($A_EVENTS as $date => $events) {
+        echo $date." = ";
+        foreach ($events as $oneE) {
+            print_r($oneE);
+            echo "<br><br>";
+        }
+        echo "<hr width=100%>";
+    }
+*/
```

Comment:
This code is in all the other files (week,month) i just added it to the day as it is very useful when adding things to this code. 
Let me know if remove it even though...